kevlar/sketch.py

```diff
@@ -10,19 +10,19 @@
 import khmer
 
 
-sketch_constructor_by_filename_extension = {
-    '.nt': khmer._Nodetable,
-    '.ng': khmer._Nodegraph,
-    '.ct': khmer._Counttable,
-    '.cg': khmer._Countgraph,
-    '.sct': khmer._SmallCounttable,
-    '.scg': khmer._SmallCountgraph,
-    '.nodetable': khmer._Nodetable,
-    '.nodegraph': khmer._Nodegraph,
-    '.counttable': khmer._Counttable,
-    '.countgraph': khmer._Countgraph,
-    '.smallcounttable': khmer._SmallCounttable,
-    '.smallcountgraph': khmer._SmallCountgraph,
+sketch_loader_by_filename_extension = {
+    '.nt':  (khmer.load_nodetable, False),
+    '.ng':  (khmer.load_nodegraph, False),
+    '.ct':  (khmer.load_counttable, False),
+    '.cg':  (khmer.load_countgraph, False),
+    '.sct': (khmer.load_counttable, True),
+    '.scg': (khmer.load_countgraph, True),
+    '.nodetable':       (khmer.load_nodetable, False),
+    '.nodegraph':       (khmer.load_nodegraph, False),
+    '.counttable':      (khmer.load_counttable, False),
+    '.countgraph':      (khmer.load_countgraph, False),
+    '.smallcounttable': (khmer.load_counttable, True),
+    '.smallcountgraph': (khmer.load_countgraph, True),
```

Comment:
These changes are unrelated to the other refactoring in this PR, and are simply keeping up with khmer changes.